modules/ossm-config-openshift-monitoring-kiali.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/observability/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-kiali_{context}"]
+= Configuring OpenShift Monitoring with Kiali
+:context: ossm-config-openshift-monitoring-kiali
+
+//Need some kind of intro
+//Possible file name may change
+//Possible assembly file may change
+//Assemblies, topic map info needs to be worked out still for 3.0, now that docs has a better idea of what is to be included for 3.0 TP1.
+
+//FIX THE STYLE THINGS ONCE DEV HAS REVIEWED/APPROVED THE CONTENT. Content came from different places, so the first priority is ensuring the steps are in the correct order, and are accurate for 3.0 TP1.
+
+The following steps show how to integrate {KialiProduct} with `user-workload monitoring`.
+
+.Prerequisites
+
+* {SMProductName} is installed.
+
+* `user-workload monitoring` is enabled. See "Enabling user-defined projects".
+
+* {KialiProduct} 1.86 is installed.
+
+.Procedure
+
+. Create a `ClusterRoleBinding` resource for Kiali:
++
+.Example `ClusterRoleBinding` configuration
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kiali-monitoring-rbac
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-monitoring-view
+subjects:
+- kind: ServiceAccount
+  name: kiali-service-account
+  namespace: istio-system
+----
+
+. Create a Kiali resource and point it to your Istio instance:
++
+.Example Kiali resource configuration
+[source,yaml]
+----
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali-user-workload-monitoring
+  namespace: istio-system
+spec:
+  external_services:
+    prometheus:
+      auth:
+        type: bearer
+        use_kiali_token: true
+      thanos_proxy:
+        enabled: true
+      url: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
+----
+
+. Get the Kiali URL from the `Route`:
++
+[source,terminal]
```

Comment:
An alternative to getting the URL from the terminal is to go through the console. Do you think this is better?
![AccessKiali](https://github.com/user-attachments/assets/bcf25f62-d98e-4c23-b968-40e9e44380b4)

Reply:
Hrm...for consistency for 3.0 TP1, let's go through the console. The instructions for Metrics + Kiali have them returning to the console, so to me, it makes sense for users to do the same here, too.

Reply:
After our Slack discussion, we've decided to keep it as is. Other options for viewing require additional info that is not part of TP 1.